lookout/style/format/benchmarks/top_repos_quality.py

```diff
@@ -148,15 +148,18 @@ class QualityReport:
     Storage for reports generated by QualityReportAnalyzer.
     """
 
-    def __init__(self, quality: Optional[str] = None, model: Optional[str] = None) -> None:
+    def __init__(self, quality: Optional[str] = None, model: Optional[str] = None,
+                 test: Optional[str] = None) -> None:
         """
         Init method.
 
         :param quality: Quality report string generated by generate_quality_report.
         :param model: Model report string generated by generate_model_report.
+        :param test: Test report string generated by generate_test_report.
         """
         self.quality = quality
         self.model = model
+        self.test = test
```

Comment:
Then we should rename it to test_report

quality -> train_report
model -> model_report

Reply:
are you sure?
then in the code, it will look like
```python
report = Report()
report. test_report  # instead of report.test
```

Reply:
Currently, it looks as `report.test`, which is even worse :D 
We can use "section" instead of "report":

```
report.train_section
report.test_section
```

Reply:
ok, I prefer `report.test_report`